### PR TITLE
Add csi-driver-manila image for cloud-provider-openstack

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml [hunk 1]
```diff
@@ -18,6 +18,14 @@ images:
...
       paths: null
   to: manila-provisioner
+- dockerfile_path: images/manila-csi-plugin/Dockerfile.openshift.ci
+  from: base
+  inputs:
+    root:
+      as:
+      - "0"
+      paths: null
+  to: manila-csi-plugin
```
Comment (smarterclayton) [inline ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml]: Does this match other csi image names?

Comment (Fedosin) [reply ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml]: I chose that name to comply with the upstream... https://github.com/kubernetes/cloud-provider-openstack/blob/master/Makefile#L271
https://hub.docker.com/r/k8scloudprovider/manila-csi-plugin

Comment (Fedosin) [reply ci-operator/config/openshift/cloud-provider-openstack/openshift-cloud-provider-openstack-master.yaml]: This is the first CSI вкшмук image in OpenShift, but I think normally they should be called `<something>-csi-driver`. 
I can change `plugin` to `driver` if you think so, but that name was taken intentionally to keep it the same as it is in the upstream.

**Comment by smarterclayton** [comment]
Hrm.  I’m wondering if this should instead be “csi-driver-manila” so that we start and enforce a consistent pattern

**Comment by Fedosin** [comment]
Done! now the image is called `csi-driver-manila`

File: core-services/image-mirroring/openshift/mapping_origin_4_5 [hunk 2]
```diff
@@ -145,6 +145,7 @@ registry.svc.ci.openshift.org/ocp/4.5:machine-config-operator-rpms quay.io/opens
 registry.svc.ci.openshift.org/ocp/4.5:machine-config-server quay.io/openshift/origin-machine-config-server:4.5 quay.io/openshift/origin-machine-config-server:4.5.0
 registry.svc.ci.openshift.org/ocp/4.5:managed-cluster-config quay.io/openshift/origin-managed-cluster-config:4.5 quay.io/openshift/origin-managed-cluster-config:4.5.0
 registry.svc.ci.openshift.org/ocp/4.5:managed-velero-operator quay.io/openshift/origin-managed-velero-operator:4.5 quay.io/openshift/origin-managed-velero-operator:4.5.0
+registry.svc.ci.openshift.org/ocp/4.5:csi-driver-manila quay.io/openshift/origin-csi-driver-manila:4.5 quay.io/openshift/origin-csi-driver-manila:4.5.0
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_5]: this is in the wrong place (needs to be alphabetical)

Comment (Fedosin) [reply core-services/image-mirroring/openshift/mapping_origin_4_5]: my bad :( fixed

**Pull request merged**